farmanager/__imageversion.txt

```diff
@@ -0,0 +1 @@
+3.0.4900
```

Comment:
The version tags could be "grepped" from the Dockerfile using eg. `ENV FAR_VERSION 3.0.4900` inside the Dockerfile. I also have to improve that for other build scripts (eg. in node folder) to reduce the number of duplicate version strings.